src/xhr.c

```diff
@@ -23,7 +23,10 @@
  */
 
 #include "curl-utils.h"
+#include "curl/curl.h"
```

Comment:
All of these includes should be covered by the existing ones, no need to add them explicitly.